### ISSUE TITLE
pylint labels incorrect no-member column in compound variables

### ISSUE THREAD

**Issue by wanderingcoder** [PyCQA/pylint#3103]
In the no-member error report for compound variable references like w.x.y.z, pylint mentions the column of the first variable/attribute in the sequence instead of the actual location of the problem. For example, if y has no member z, pylint says the problem is in column 0 (w) and not column 4 (y).

This isn't as much of a big deal when you're running it on the command line, but some IDEs (VSCode in my case) use the column identifier to highlight which part of the line is at fault, and it can be very misleading. 

They pointed me in your direction since they're relying on the column number output from pylint (see https://github.com/microsoft/vscode-python/issues/7327).

### Steps to reproduce
```python
class TestMe:
    def __init__(self):
        self.item = {'akey': 42}
        self.again = self

x = TestMe()
### pylint error message here is
###    testme.py:11:6: E1101: Instance of 'dict' has no 'akey' member (no-member)
### The problem is with `x.item`, but pylint shows the column for `x`
print(x.item.akey > 2)

print(x.again.item.doesnotexist)
```

### Current behavior
```
$ pylint testme.py  -rn -sn
************* Module testme
testme.py:10:6: E1101: Instance of 'dict' has no 'akey' member (no-member)
testme.py:12:6: E1101: Instance of 'dict' has no 'doesnotexist' member (no-member)
```

### Expected behavior
```
$ pylint testme.py  -rn -sn
************* Module testme
testme.py:10:8: E1101: Instance of 'dict' has no 'akey' member (no-member)
testme.py:12:14: E1101: Instance of 'dict' has no 'doesnotexist' member (no-member)
```

### pylint --version output
pylint 2.3.1
astroid 2.2.5
Python 3.7.4 (default, Jul  9 2019, 18:13:23)
[Clang 10.0.1 (clang-1001.0.46.4)]

**Comment by wanderingcoder** [comment]
Created Python issue https://bugs.python.org/issue38166

**Comment by wanderingcoder** [comment]
A [comment on the Python issue](https://bugs.python.org/issue38166#msg352467) indicates that to be more specific about where the problem lies, you'd probably need to find attr='item', then use (col_offset, col_offset+end_col_offset) to indicate the sequence that includes the attribute (i.e. columns 0-5).

**Comment by wanderingcoder** [comment]
Reposting the "not a bug" comment from the issue I filed with CPython: https://bugs.python.org/issue38166#msg352941

> As Serhiy explained, there is no bug here, _potentially_ we could change AST so that attribute name (i.e. "b" in "a.b") is wrapped in a special node holding the position info (similar to ``ast.arg``), but it is a breaking change and is not worth it. You can re-open your issue on PyLint tracker.
